src/BUILD

```diff
@@ -123,12 +123,15 @@ JAVA_TOOLS = [
     "//src/java_tools/buildjar/java/com/google/devtools/build/buildjar/jarhelper:srcs",
     "//src/tools/singlejar:embedded_tools",
     "//src/java_tools/import_deps_checker/java/com/google/devtools/build/importdeps:embedded_tools",
-    "//third_party/ijar",
+    # TODO(#6316): Move this target to the remote java tools archive."
     "//third_party/ijar:zipper",
-    "//third_party/ijar:embedded_tools",
     "//third_party/java/jacoco:srcs",
 ] + select({
     "//src/conditions:arm": [],
+    "//src/conditions:windows": [
+        # TODO(#6316): Remove this after the Java tools archive for Winodws is released.
+        "//third_party/ijar",
```

Comment:
Why do you do this only for Windows?

Reply:
You are right, this was meant for any other non-linux platforms not only Windows. Done.